package.json

```diff
@@ -16,7 +16,7 @@
     "is-composer-package-required": "./bin/is-composer-package-required"
   },
   "scripts": {
-    "lint": "eslint *.js"
+    "lint": "./node_modules/.bin/eslint ./src/*.js"
```

Comment:
`npm` helpfully adds `node_modules/.bin` to the path so it does not need to be specified.